explainaboard/info.py

```diff
@@ -191,13 +184,13 @@ def copy(self) -> "SysOutputInfo":
 
 @dataclass
 class OverallStatistics:
-    sys_info: SysOutputInfo = None
```

Comment:
I think `sys_info` is not actually optional.

Reply:
Fixed: ba85ab9